modules/apps/commerce/commerce-frontend-impl/build.gradle

```diff
@@ -11,7 +11,6 @@ dependencies {
 	compileOnly group: "org.apache.cxf", name: "cxf-rt-frontend-jaxrs", version: "3.4.10"
 	compileOnly group: "org.apache.cxf", name: "cxf-rt-rs-extension-providers", version: "3.4.10"
 	compileOnly group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
-	compileOnly group: "org.apache.struts", name: "struts-core", version: "1.3.10"
```

Comment:
the ticket says 
_Please remove the dependency on Struts 1.x or upgrade to Struts 2.x._
not to remove this dependency

Reply:
@andrea-ale-sbarra , I am confused here. The only time we are using Struts is to do the session remove. We are no longer using Struts for this. The correct implementation is to remove. The removal aligns with a previous LPS ticket (See linked ticket on COMMERCE-9781 and my comment there).

Sincerely,
Brian I. Kim

Reply:
I linked the relevant commits directly:

See: https://github.com/liferay/liferay-portal/commit/a1db809bb052162dcb71fb218db47c1bb85a6789
and: https://github.com/liferay/liferay-portal/commit/0f6e75203fd4fd674807e92cfe011c1f560f2ea6